pkg/operator/keys/list.go

```diff
@@ -91,6 +101,53 @@ func GetPubKey(keyStoreFile string) (string, error) {
 	}
 }
 
+func GetOperatorIdFromPubKey(pubKey string) (string, error) {
+	// From:
```

Comment:
@fyInALT was this meant to be included?

Reply:
Not need include, i just change comments more clear.